BLSQ/iaso
# IA-1556: use components from blsq-comp

Budget tool was using components that  could be reused and would fit better in bluesquare-components

## Self proof reading checklist


## Changes

- Deleted `CustomInput` , `FilesUpload`, `SimpleModal`, `ModalWithButtons`, `ConfirmCancelButtons` and `ConfirmCancelModal` 
- Replaced with imports from bluesquare-components
- Removed dep to `react-dropzone`


## How to test
- Go to Polio> Budget
- Select a campaign
- click "Add step"
- you should be able to upload files and the modal should work as before

## Print screen / video

NA

## Notes
Comes with https://github.com/BLSQ/bluesquare-components/pull/79
⚠️ package and package-lock should be updated before merging
